### PR TITLE
Allow user to receive notification

### PR DESCRIPTION
**Pull request by takai**
Allow user to receive a notification when the job is finished.

## UI

screen shot 2017-02-13 at 19 23 57

## Example
screen shot 2017-02-13 at 19 23 12

### CODE REVIEW

File: app/assets/javascripts/kuroko2/job_instances.js [hunk 1]
```diff
@@ -3,11 +3,28 @@
...
   var logIntervalId;
+  var notifyIfNeeded = function (status, name) {
+    if (!('Notification' in window)) {
+      return;
+    }
+
+    if (Notification.permission === 'granted' && Cookies.get('notification') === 'on') {
+      var notification = new Notification(
+        "[" + status + "] " + name,
+        {"icon": window.location.origin + "/assets/kuroko2/kuroko-logo-" + status.toLowerCase() + ".png"}
```
Comment (eisuke) [inline app/assets/javascripts/kuroko2/job_instances.js]: It is not necessarily to mount kuroko2 engine on root.
How about using the rails generated path or URL in any data attribute?

Comment (takai) [inline app/assets/javascripts/kuroko2/job_instances.js]: Assets within an engine are put into /assets path. This code will work.

Comment (eisuke) [inline app/assets/javascripts/kuroko2/job_instances.js]: Oops... I'm sorry, what I pointed out is mistake 😢 
But the same situation will occur using `config.assets.prefix` or` config.action_controller.asset_host`, right?

Comment (takai) [inline app/assets/javascripts/kuroko2/job_instances.js]: Your are right. I will change it.

Comment (takai) [inline app/assets/javascripts/kuroko2/job_instances.js]: Fixed 7ecc693

**Pull request merged**